boot_services/src/boot_services.rs

```diff
@@ -262,19 +262,51 @@ pub trait BootServices: Sized {
         nb_pages: usize,
     ) -> Result<usize, efi::Status>;
 
+    /// Frees memory pages.
+    ///
+    /// UEFI Spec Documentation:
+    /// <a href="https://uefi.org/specs/UEFI/2.10/07_Services_Boot_Services.html#efi-boot-services-freepages" target="_blank">

```

Comment:
Please use markdown style links instead of directly using html style. `rustdoc` will generate the links for you.

[<name>](<link>)

Reply:
Thanks, are you saying instead of 

/// Create an event.
/// 
/// <a href="https://uefi.org/specs/UEFI/2.10/07_Services_Boot_Services.html#efi-boot-services-createevent" target="_blank">
///   7.1.1. EFI_BOOT_SERVICES.CreateEvent()
/// </a>

it should be updated to following, in all methods? Did a quick change and validate it opens the link in same browser window instead of target="_blank" causing it to open in a new window. If that is desired, please let me know.

/// Create an event.
/// 
/// [UEFI Spec Documentation: 7.1.1. EFI_BOOT_SERVICES.CreateEvent()](https://uefi.org/specs/UEFI/2.10/07_Services_Boot_Services.html#efi-boot-services-createevent)

Reply:
Hello, yes my preference, and the canonical rust documentation way, is to create links using the markdown method. It may just be github, but even your first link using the href opens in this same window. Regardless, I don't mind that happening (opening in the same window) due to using the markdown method, because a user can simply control-click or middle-click to open it in a new window.

Reply:
And yes, for all methods please :)

Reply:
Sounds good, thanks for clarifying.